sdk/servicebus/service-bus/samples/javascript/advanced/processMessageFromDLQ.js

```diff
@@ -23,7 +24,7 @@ const queueName = process.env.QUEUE_NAME || "<queue name>";
 
 const sbClient = new ServiceBusClient(connectionString);
 
-async function main() {
+export async function main() {
```

Comment:
Why export?

Reply:
Let me check on this. I wanted to minimize differences between the TS and JS in the future but I can't recall why it's like that in the TypeScript.

Reply:
I think it was required for us to run the samples in CI. Check with @willmtemple

Reply:
That is what it is. For now I'm going to leave it in place just to reduce the arbitrary number of diffs when syncing TS and JS.